cnd/src/network/comit/swaps.rs

```diff
@@ -310,20 +310,25 @@ mod tests {
     use crate::{
         asset::{self, ethereum::FromWei},
         identity,
-        network::LocalData,
+        network::comit::LocalData,
```

Comment:
This change is unnecessary, network.rs exports everything in the comit module (`pub use comit::*`).